### PR TITLE
Ckeditor integration

### PR DESCRIPTION
**Pull request by DevMike**
http://bortovik.worksection.com/project/33775/715943/

### CODE REVIEW

File: app/admin/car_brands.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@
   form do |f|
     f.inputs "Details" do
       f.input :name
+      f.input :description, :as => :ckeditor
```
Comment (kont-noor) [inline app/admin/car_brands.rb:26]: use ruby 1.9 hashes

File: app/admin/car_models.rb [hunk 2]
```diff
@@ -26,6 +26,7 @@
   form do |f|
```
Comment (kont-noor) [inline app/admin/car_models.rb:26]: can we move form to separate partial?

File: app/admin/car_modifications.rb [hunk 3]
```diff
@@ -31,6 +31,7 @@
   form do |f|
```
Comment (kont-noor) [inline app/admin/car_modifications.rb:31]: the same question

File: app/assets/stylesheets/active_admin.css.scss [hunk 4]
```diff
@@ -27,3 +27,8 @@
 // .active_admin       applies to any Active Admin namespace
 // .admin_namespace    applies to the admin namespace (eg: /admin)
 // .other_namespace    applies to a custom namespace named other (eg: /other)
+
+.cke_skin_kama .cke_wrapper {
+  background-color: transparent!important;
+  background-image: none!important;
```
Comment (kont-noor) [inline app/assets/stylesheets/active_admin.css.scss]: just use background: none instead of these 2 lines

File: app/models/ckeditor/asset.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class Ckeditor::Asset < ActiveRecord::Base
+  include Ckeditor::Orm::ActiveRecord::AssetBase
+              
+  delegate :url, :current_path, :size, :content_type, :filename, :to => :data
```
Comment (kont-noor) [inline app/models/ckeditor/asset.rb:4]: ruby 1.9. hash

File: app/models/ckeditor/attachment_file.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+class Ckeditor::AttachmentFile < Ckeditor::Asset
+  mount_uploader :data, CkeditorAttachmentFileUploader, :mount_on => :data_file_name
```
Comment (kont-noor) [inline app/models/ckeditor/attachment_file.rb:2]: also

File: app/admin/car_brands.rb (hunk 1 above)
Comment (DevMike) [inline app/admin/car_brands.rb:26]: these ones are generated automatically, don't see sense to change them in this case

File: app/admin/car_models.rb (hunk 2 above)
Comment (DevMike) [inline app/admin/car_models.rb:26]: don't see sense for that - the form is small

**Pull request merged**